app/_locales/de/messages.json

```diff
@@ -958,7 +958,7 @@
     "message": "Hierdurch werden ein neues Wallet und eine mnemonische Phrase erzeugt"
   },
   "tips": {
-    "message": "Trinkgelder"
+    "message": "Tipps"
```

Comment:
Google Translate confirms this as "tips"

Reply:
Correct. "Trinkgelder" would be like "tips" that you give to the waiter/waitress